hooks/tk-multi-publish2/nuke/publish.py

```diff
@@ -548,17 +560,19 @@ def _write_node_path_duplicacy(self, item):
         if node_path in all_paths:
             duplicate_path_node = [key for (key, value) in item.parent.properties['write_node_paths_dict'].items()
                                    if value == node_path]
-            self.logger.error("Duplicate output path.",
-                              extra={
-                                  "action_show_more_info": {
-                                      "label": "Show Info",
-                                      "tooltip": "Show node(s) with identical output path",
-                                      "text": "Following node(s) have same output path as {}:\n\n{}".
-                                      format(node_name, '\n'.join(duplicate_path_node))
+
+            if node_name not in duplicate_path_node:
```

Comment:
This seems risky. Even in node_name is present in the list, we could have other nodes which have duplicate paths, which should still make validation return False.
Perhaps we can delete the node_name if it exists, and then check whether there are any nodes left in the list?
Also, maybe rename `duplicate_path_node` to `duplicate_path_nodes`.

Reply:
The logic here is that each unique path will be associated with the first node having that path. If there are other nodes having the same path, then they will be compared against that first node. That's why the logic `if node_name not in duplicate_path_node:`. 
The error message is a bit misleading. Should have fixed that.

But yea its better to show list of all nodes sharing the path. Fixing the logic.